src/runtime-rs/Makefile

```diff
@@ -170,11 +170,7 @@ DEFMSIZE9P := 8192
 DEFVFIOMODE := guest-kernel
 ##VAR DEFSANDBOXCGROUPONLY=<bool> Default cgroup model
 DEFSANDBOXCGROUPONLY ?= false
-DEFSANDBOXCGROUPONLY_DB ?= true
-DEFSANDBOXCGROUPONLY_FC ?= true
 DEFSTATICRESOURCEMGMT ?= false
```

Comment:
```
greg@bahia:[feat_fix_qemu_after_8070]kata-containers$ git grep DEFSTATICRESOURCEMGMT src/runtime-rs
src/runtime-rs/Makefile:DEFSTATICRESOURCEMGMT ?= false
src/runtime-rs/Makefile:    DEFSTATICRESOURCEMGMT_DB := false
src/runtime-rs/Makefile:    DEFSTATICRESOURCEMGMT_CLH := true
src/runtime-rs/Makefile:    DEFSTATICRESOURCEMGMT_QEMU := true
src/runtime-rs/Makefile:    DEFSTATICRESOURCEMGMT_FC = true
src/runtime-rs/Makefile:USER_VARS += DEFSTATICRESOURCEMGMT
src/runtime-rs/Makefile:USER_VARS += DEFSTATICRESOURCEMGMT_DB
src/runtime-rs/Makefile:USER_VARS += DEFSTATICRESOURCEMGMT_FC
src/runtime-rs/Makefile:USER_VARS += DEFSTATICRESOURCEMGMT_CLH
src/runtime-rs/Makefile:USER_VARS += DEFSTATICRESOURCEMGMT_QEMU
src/runtime-rs/config/configuration-cloud-hypervisor.toml.in:static_sandbox_resource_mgmt=@DEFSTATICRESOURCEMGMT_CLH@
src/runtime-rs/config/configuration-dragonball.toml.in:static_sandbox_resource_mgmt=@DEFSTATICRESOURCEMGMT_DB@
src/runtime-rs/config/configuration-qemu-runtime-rs.toml.in:static_sandbox_resource_mgmt=@DEFSTATICRESOURCEMGMT_QEMU@
src/runtime-rs/config/configuration-rs-fc.toml.in:static_sandbox_resource_mgmt=@DEFSTATICRESOURCEMGMT_FC@
```

Unless I'm missing something, it seems that `DEFSTATICRESOURCEMGMT` isn't used anymore. Maybe add a commit to fully remove it ?

Reply:
correct, pushed a new commit that removes the generic vars and keeps only the per-hypervisor ones.